onnxruntime/core/providers/xnnpack/math/gemm.h

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/xnnpack/xnnpack_kernel.h"
+#include "core/framework/allocator.h"
+#include "core/providers/cpu/math/gemm_base.h"
+#include "core/providers/xnnpack/detail/utils.h"
+#include "xnnpack.h"
+#include "core/common/common.h"
+#include "core/util/math.h"
```

Comment:
Could we move some headers onto `.cc` files?

Reply:
I think I addressed this issue in one of my last commits - please the latest version